tests/Behat/Context/Setup/MollieContext.php

```diff
@@ -82,13 +135,32 @@ public function theStoreHasAPaymentMethodWithACodeAndMollieSubscriptionPaymentGa
         $paymentMethod->getGatewayConfig()->setConfig([
             'api_key' => 'test',
             'payum.http_client' => '@bitbag_sylius_mollie_plugin.mollie_api_client',
-            'times' => 1,
-            'interval' => '1 months',
+            'api_key_test' => $this->mollieApiKeyTest,
+            'profile_id' => $this->mollieProfileId,
+            'environment' => null,
         ]);
 
         $this->paymentMethodManager->flush();
     }
 
+    private function loadMolliePaymentMethods(GatewayConfigInterface $gatewayConfig): void
+    {
+        $this->mollieMethodsResolver->createForGateway($gatewayConfig);
+
+        $this->molliePaymentMethodPurifier->removeAllNoLongerSupportedMethods();
+    }
+
+    private function enableAllMolliePaymentMethods(): void
+    {
+        $molliePaymentMethods = $this->mollieConfigurationRepository->findAll();
+
+        /** @var MollieGatewayConfigInterface $molliePaymentMethod */
+        foreach ($molliePaymentMethods as $molliePaymentMethod) {
+            $molliePaymentMethod->enable();
+            $molliePaymentMethod->setCountryRestriction(MollieGatewayConfigInterface::ALL_COUNTRIES);
```

Comment:
How about splitting the long line into smaller ones?

Reply:
Done